dev/Scroller/APITests/ScrollerViewChangeTests.cs

```diff
@@ -37,7 +37,7 @@
 
 namespace Windows.UI.Xaml.Tests.MUXControls.ApiTests
 {
-    partial class ScrollerTests
+    partial class ScrollerTests : ApiTestBase
```

Comment:
The class `ScrollerTests` already is becoming a subclass of "ApiTestBase" in "ScrollerAnchoringTests.cs". For consistency only one file or all files should have "ApiTestBase" specified as its base class.

Reply:
Thanks, good catch

---
In reply to: [358435871](https://github.com/microsoft/microsoft-ui-xaml/pull/1760#discussion_r358435871) [](ancestors = 358435871)